priv/test_files/with_warnings/src/b_module.erl

```diff
@@ -0,0 +1,7 @@
+-module(b_module).
+
+-export([my_fun/2]).
+
+%% _Arg2 is unused
+my_fun(Arg1, Arg2) ->
```

Comment:
We also need a test function where some argument is ignored in one clause, but not in all of them.
Maybe 2 or 3 functions, actually… to distinguish between `_` and `_Arg2` (as mentioned above).